driver-opentelemetry/src/main/java/com/datastax/driver/opentelemetry/OpenTelemetryTracingInfoFactory.java

```diff
@@ -7,31 +7,41 @@
 import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Context;
 
+import static com.datastax.driver.opentelemetry.PrecisionLevel.NORMAL;
```

Comment:
I think you should do 
```
import com.datastax.driver.opentelemetry.PrecisionLevel;
```
and then use ```PrecisionLevel.NORMAL``` instead of ```NORMAL```.